tensorflow/lite/kernels/add_n_test.cc

```diff
@@ -88,6 +88,77 @@ TEST(IntegerAddNOpModel, AddMultipleTensors) {
   EXPECT_THAT(m.GetOutput(), ElementsAreArray({-9, -1, 11, 11}));
 }
 
+class QuantizedAddNOpModel : public BaseAddNOpModel {
+ public:
+   using BaseAddNOpModel::BaseAddNOpModel;
+
+  template <typename integer_dtype>
+  std::vector<float> GetDequantizedOutput() {
+    return Dequantize<integer_dtype>(ExtractVector<integer_dtype>(output_),
+                                     GetScale(output_), GetZeroPoint(output_));
+  }
+};
+
+// for quantized Add, the error shouldn't exceed step
+float GetTolerance(float min, float max) {
```

Comment:
Do you have a reason to reuse the method from "add.cc"? I wonder if this is still true for AddN since there could be more than 2 inputs and the tolerance (theoretical limits on error introduced by quantization) depends on N.